assets/src/edit-story/components/panels/textStyle.js

```diff
@@ -0,0 +1,287 @@
+            onChange={(value) => {
+              const currentFontWeights = getFontWeight(value);
+              const currentFontFallback = getFontFallback(value);
+              const fontWeightsArr = currentFontWeights.map(
+                ({ thisValue }) => thisValue
+              );
+              const newFontWeight =
+                fontWeightsArr && fontWeightsArr.includes(state.fontWeight)
+                  ? state.fontWeight
+                  : 400;
+              setState({
+                ...state,
+                fontFamily: value,
+                fontWeight: parseInt(newFontWeight),
+                fontWeights: currentFontWeights,
+                fontFallback: currentFontFallback,
+              });
+            }}
+          />
+        )}
+      </Row>
+      <Row>
+        <SelectMenu
+          ariaLabel={__('Font style', 'web-stories')}
+          options={fontStyles}
+          isMultiple={fontStyle === ''}
+          value={state.fontStyle}
+          onChange={(value) => setState({ ...state, fontStyle: value })}
+        />
+        <Space />
+        <BoxedNumeric
+          ariaLabel={__('Font size', 'web-stories')}
+          value={state.fontSize}
+          isMultiple={fontSize === ''}
+          min={MIN_FONT_SIZE}
+          max={MAX_FONT_SIZE}
+          flexBasis={58}
+          textCenter
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
+          }
+        />
+      </Row>
+      <Row>
+        {/* TODO: Add vertical offset logic */}
+        <ExpandedNumeric
+          ariaLabel={__('Vertical offset', 'web-stories')}
+          value={state.fontSize}
+          suffix={<VerticalOffset />}
+          isMultiple={fontSize === ''}
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
+          }
+        />
+        <Space />
+        {/* TODO: Add horizontal offset logic */}
+        <ExpandedNumeric
+          ariaLabel={__('Horizontal offset', 'web-stories')}
+          value={state.fontSize}
+          suffix={<HorizontalOffset />}
+          isMultiple={fontSize === ''}
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
+          }
+        />
+      </Row>
+      <Row>
+        <SelectMenu
+          label={__('Alignment', 'web-stories')}
+          options={alignmentOptions}
+          isMultiple={'' === textAlign}
+          value={state.textAlign}
+          onChange={(value) => setState({ ...state, textAlign: value })}
+        />
+      </Row>
+      <Row>
+        <Label>{__('Text', 'web-stories')}</Label>
+        <Color
+          value={state.backgroundColor}
+          onChange={(value) => setState({ ...state, backgroundColor: value })}
+          opacity={1}
+        />
+      </Row>
+      <Row>
+        <Label>{__('Padding', 'web-stories')}</Label>
+        <BoxedNumeric
+          suffix={_x('H', 'The Horizonatal padding', 'web-stories')}
```

Comment:
```suggestion
          suffix={_x('H', 'The Horizontal padding', 'web-stories')}
```

Reply:
Thank you for your point @swissspidy . Fixed typo